src/handlers/canonicalFieldsHandlers.ts

```diff
@@ -63,9 +64,7 @@ const postCanonicalFieldBodySchema: JSONSchemaType<Omit<CanonicalField, 'created
     shortCode: {
       type: 'string',
     },
-    dataType: {
-      type: 'string',
-    },
+    dataType: jsonSchemaObject,
```

Comment:
Oh - I don't know if there's an existing, exported schema schema that AJV provides to allow this kind of reference.

Reply:
There must still be a better Ajv-ish way but I am glad it's not just me that couldn't easily find it.

Reply:
I wish https://ajv.js.org/api.html#ajv-validateschema-schema-object-boolean took the missing `JsonSchema` type instead of an `object`.